src/controllers/beneficial.owner.individual.controller.ts

```diff
@@ -1,11 +1,31 @@
-import { Request, Response } from "express";
+import { NextFunction, Request, Response } from "express";
+import { getApplicationData, prepareData, setApplicationData } from "../utils/application.data";
+import { ApplicationData, ApplicationDataType, beneficialOwnerIndividualType } from "../model";
 
 import { logger } from "../utils/logger";
 import * as config from "../config";
 
 export const get = (req: Request, res: Response) => {
-  logger.info(`GET THE INDIVIDUAL BENEFICIAL OWNER PAGE`);
+  logger.info(`GET ${config.BENEFICIAL_OWNER_INDIVIDUAL_PAGE}`);
```

Comment:
i'd have a logger.debug here, seems more of a debug message then real useful info.